Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -613,18 +613,21 @@ def __exit__(self, type, value, traceback):
 # TODO: more consistency checks:
 # - is there gantry tilt?
 # - are the orientations the same for all slices?
-def getSortedImageFiles(filePaths, epsilon=0.01):
-    """ Sort DICOM image files in increasing slice order (IS direction) corresponding to a series
+def getSortedImageFiles(filePaths: list[str], epsilon: float=0.01) -> tuple[list[str], dict[str, str], str]:
+    """Sort DICOM image files in increasing slice order (IS direction) corresponding to a series
 
-        Use the first file to get the ImageOrientationPatient for the
-        series and calculate the scan direction (assumed to be perpendicular
-        to the acquisition plane)
+    Use the first file to get the ImageOrientationPatient for the
+    series and calculate the scan direction (assumed to be perpendicular
+    to the acquisition plane)
 
-        epsilon: Maximum difference in distance between slices to consider spacing uniform
+    :param filePaths : Paths of the local DICOM files to sort.
+    :param epsilon: Maximum difference in distance between slices to consider spacing uniform.
+
+    :return: Tuple of (files, distances, warningText)
```

Comment:
It would be nice to have the types here to avoid issues like the one James pointed out.

Reply:
Issue of @jamesobutler has just been addressed.

Documentation of the return type has been addressed in the second commit by introducing the corresponding type annotation.